examples/typescript/http-client-pool/src/worker.ts

```diff
@@ -11,7 +11,7 @@ class HttpClientWorker extends ThreadWorker<WorkerData, WorkerResponse> {
     super({
       node_fetch: async (workerData?: WorkerData) => {
         const response = await nodeFetch(
-          (workerData as WorkerData).input as URL | NodeFetchRequestInfo,
+          (workerData as WorkerData).input,
```

Comment:
Not sure about this change, but it doesn't come from `biome`. I checked and it seems that `ts-standard` does that.

Reply:
Should be fine, latest @typescript/eslint version properly remove unneeded type casting. 
You can check if typescript examples are still build with the ugly `build.sh` shell script in the directory.

Reply:
Unfortuantely it doesn't build. What do you suggest to do? I'd like to have the PR merged you can keep having the package up to date :)

Reply:
> Unfortuantely it doesn't build. What do you suggest to do? I'd like to have the PR merged you can keep having the package up to date :)

I will merge it with the fixes (and maybe biome configuration changes) in https://github.com/poolifier/poolifier/pull/1129. 
To keep the original authoring, I prefer to merge that one first once it contains the fixes. 

And just put back the type casting to fix the example build.